index.html

```diff
@@ -118,84 +133,92 @@ <h2 class="projtxt">Project Name Goes Here</h2>
-        <div class="maintitle">
-            <h1 class="mtitle" >Get started</h1>
-        </div>
-        <div class="maintxt">
-            <p class="mtext" >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
-        </div>
-        <form action="https://formspree.io/f/mayvvvnd" method="POST">
-            <ul class="contactform">
-                <li>
-                <label for="fname">Your name</label>
-                    <input type="text" name="name" id="fname" placeholder="Jhon Doe" required="" maxlength="30" pattern="([A-ZΆ-ΫÀ-ÖØ-Þ][A-ZΆ-ΫÀ-ÖØ-Þa-zά-ώß-öø-ÿ]{1,19} ?){1,10}">
-                </li>
-                <li>
-                <label for="email-address">Your email</label>
-                    <input type="email" name="email" id="email-address" placeholder="email@domain.com" required="" pattern="(?!(^[.-].*|[^@]*[.-]@|.*\.{2,}.*)|^.{254}.)([a-zA-Z0-9!#$%&'*+\/=?^_`{|}~.-]+@)(?!-.*|.*-\.)([a-zA-Z0-9-]{1,63}\.)+[a-zA-Z]{2,15}">
-                </li>
-                <li>
-                <textarea rows="4" name="message" id="message" placeholder="Enter your message here..." required="" maxlength="500"></textarea>
-                </li>
-            </ul>
-            <button class="collabbutton" type="submit">Start collaboration</button>
-        </form>
+    <section class="contact" id="contact">
+        <div class="topimg"></div>
+        <div class="contactctn">
+            <div class="maintitle">
+                <h1 class="mtitle" >Get started</h1>
+            </div>
+            <div class="maintxt">
+                <p class="mtext" >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            </div>
+            <form action="https://formspree.io/f/mayvvvnd" method="POST">
+                <ul class="contactform">
+                    <li>
+                    <label for="fname">Your name</label>
+                        <input type="text" name="name" id="fname" placeholder="Jhon Doe" required="" maxlength="30" pattern="([A-ZΆ-ΫÀ-ÖØ-Þ][A-ZΆ-ΫÀ-ÖØ-Þa-zά-ώß-öø-ÿ]{1,19} ?){1,10}">
+                    </li>
+                    <li>
+                    <label for="email-address">Your email</label>
+                        <input type="email" name="email" id="email-address" placeholder="email@domain.com" required="" pattern="(?!(^[.-].*|[^@]*[.-]@|.*\.{2,}.*)|^.{254}.)([a-zA-Z0-9!#$%&'*+\/=?^_`{|}~.-]+@)(?!-.*|.*-\.)([a-zA-Z0-9-]{1,63}\.)+[a-zA-Z]{2,15}">
+                    </li>
+                    <li>
+                    <textarea rows="4" name="message" id="message" placeholder="Enter your message here..." required="" maxlength="500"></textarea>
+                    </li>
+                </ul>
+                <button class="collabbutton" type="submit">Start collaboration</button>
+            </form>
+        </div>
+        <div class="bottomimg"></div>
     </section>
```

Comment:
- [ ] [OPTIONAL] Please make changes! The `name` and `last name` input text share the same row. But you miss adding `last name` input after the `name` input. This will be good for your next milestones!

Reply:
Thanks a lot. Will do.